tools/docker/start.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+sudo sh -c 'echo "
+127.0.0.1	web-platform.test
+127.0.0.1	www.web-platform.test
+127.0.0.1	www1.web-platform.test
+127.0.0.1	www2.web-platform.test
+127.0.0.1	xn--n8j6ds53lwwkrqhv28a.web-platform.test
+127.0.0.1	xn--lve-6lad.web-platform.test
+0.0.0.0	nonexistent-origin.web-platform.test" >> /etc/hosts'
+
+cd web-platform-tests
+git pull --depth=1
+
+# Install Chome unstable
```

Comment:
In https://github.com/w3c/web-platform-tests/pull/8838 I tried to banish "unstable" everywhere except package name, can you call it "Chrome dev"?